README.md

```diff
@@ -3,7 +3,7 @@
 Generates fake data for testing or populating a development database.
```

Comment:
com.github.blocoio:faker:1.2.9, I know you will update this after, but I guess this goes to 2.0.0...? or 1.3? Just a question :D

Reply:
and the increase readme with the get started and examples. let me know if you want help